mass_mailing_event/models/event_state.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class EventState(models.Model):
+    _name = 'event.state'
+
+    name = fields.Char(required=True)
```

Comment:
Because selection of event states for excluding is a many2many field, similar to partner tags. None can create/update/delete items in this model except admin or via XML